src/xc_integrator/xc_data/device/xc_device_data.hpp

```diff
@@ -60,7 +75,11 @@ struct required_term_storage {
   bool grid_vrho     = false;
   bool grid_vgamma   = false;
 
+  bool grid_den_uks  = false;
```

Comment:
This should be derivable from `grid_den` and `ks_scheme`

Reply:
This one was a bit of a puzzle.... It is set later on in the required_term_storage explicit function using the is_xc and ks_scheme flags. When GGAs are implemented this section may get hairy, so I'm prioritizing other items atm since it works at least for testing.

Reply:
Sure, my point is that it shouldn't be a variable in an of itself - it should _**always**_ be derived from the aforementioned. Handling variables that are from a Cartesian product of selections, you can see that things grow quickly (and the resulting code becomes more complicated as all cases necessarily have to handled)